CI/travis/before_install_linux

```diff
@@ -14,6 +14,12 @@ install_sphinx() {
 }
 
 handle_centos() {
+	# needed for man2html and a few other popular tools
+	yum search epel-release
+	yum info epel-release
+	yum -y install epel-release
+	yum -y update
```

Comment:
IIRC `yum -y update` == `apt-get update && apt-get upgrade`
i.e. it updates all installed packages;
if it's not needed [upgrade existing packages], i wouldn't do it; the time this adds to the build-run may be just a few seconds, but i'm sometimes a bit picky about build-times

for the CentOS dockers [here], it won't make much difference whether we add it or not;
they come pretty empty, so after adding this repo, we would typically install the latest versions of packages;

Reply:
It was in the all the steps I found to properly install epel-releases.

I will take it out and see if everything still all builds properly.

Reply:
removed on latest forced push